src/trufont/util/drawing.py

```diff
@@ -440,9 +440,14 @@ def drawLayerPoints(
             # others
             prev = points[-1]
             breakHandle = path.open
+            quadraticPath = any(point.type == "qcurve" for point in points)
             for point in points:
                 isOff = point.type is None
-                if not breakHandle and isOff != (prev.type is None):
+                if (
+                    not breakHandle
+                    and isOff != (prev.type is None)
+                    and not quadraticPath
+                ):
```

Comment:
do we even need to change the code that works for curve?

Reply:
ok I see the commit is named "hide handles". why hide? can you add a picture of what you want to do with handles?